examples/adc_oneshot.rs

```diff
@@ -0,0 +1,50 @@
+//! ADC oneshot example, reading a value form a pin and printing it on the terminal
+//! requires ESP-IDF v5.0 or newer
+
+use esp_idf_sys::{self as _}; // If using the `binstart` feature of `esp-idf-sys`, always keep this module imported
```

Comment:
`esp_idf_hal::sys::{self as _};`

Reply:
With that I get 
```
warning: unused import: `self as _`
 --> examples/adc_oneshot.rs:4:24
  |
4 | use esp_idf_hal::sys::{self as _}; // If using the `binstart` feature of `esp-idf-sys`, always keep this module imported
  |                        ^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
```

Reply:
Remove it then.

Reply:
so `use esp_idf_sys as _;` is not required any more? (all the other examples have it)

Reply:
Not sure. If CI does not fail, you don't need it.

Reply:
ok - will try without